fMRIVolume/GenericfMRIVolumeProcessingPipeline.sh

```diff
@@ -236,7 +245,7 @@ Compliance="HCPStyleData"
 ComplianceMsg=""
 
 if [ "${DistortionCorrection}" = 'NONE' ]; then
-  ComplianceMsg+=" --dcmethod=NONE"
+  ComplianceMsg+=" --dcmethod=NONE WARNING: The fMRIVolume pipeline is being run without appropriate distortion correction of the fMRI image. This is NOT RECOMMENDED under normal circumstances. We will attempt 6 DOF FreeSurfer BBR registration of the distorted fMRI to the undistorted T1w image. Distorted portions of the fMRI data will not align with the cortical ribbon. In HCP data 30% of the cortical surface will be misaligned by at least half cortical thickness and 10% of the cortical surface will be completely misaligned by a full cortical thickness. At a future time, we may be able to add support for fieldmap-less distortion correction; however, no extant software package or pipelines have successfully demonstrated clear improvement from fieldmap-less distortion correction when compared with a gold standard fieldmap-based correction."
```

Comment:
I believe we want this warning to ALWAYS be displayed when `DistortionCorrection` is `NONE`, so it should be a separate log_Warn right here, not combined with the message string.  The only drawback is that this will put this warning before the abort message.

Reply:
I changed it so that it is displayed separately and only when legacy mode is chosen.